dll/win32/cryptui/main.c

```diff
@@ -7399,6 +7417,12 @@ static LRESULT CALLBACK select_cert_dlg_proc(HWND hwnd, UINT msg, WPARAM wp, LPA
         }
         }
         break;
+#ifdef __REACTOS__
+    case WM_DESTROY:
+        data = (struct SelectCertData *)GetWindowLongPtrW(hwnd, DWLP_USER);
+        HeapFree(GetProcessHeap(), 0, data);
```

Comment:
```suggestion
        free_certs(GetDlgItem(hwnd, IDC_SELECT_CERTS));
        ImageList_Destroy(data->imageList);
        HeapFree(GetProcessHeap(), 0, data);
```
and move this WM_DESTROY below WM_INITDIALOG.